blue_scribe.tex

```diff
@@ -12,17 +12,100 @@
 \maketitle
 
 \section*{Introduction}
+For our final project in ECE 3710, we are using our microcontroller and a PC to implement a 2D laser engraver. To do this, we implemented code to scan an image file and break the image down into motor and laser instructions. Following this, the PC would send these instructions serially to the microcontroller via USB. The microcontroller uses the UART to receive and parse the instructions so they can be turned into motor and laser commands. The result is a fully functional laser engraver.
+
 \section*{Scope}
+This document discusses the design overview, design details, and testing methods for our laser engraver. Add more to this but im not sure what to say
+
 \section*{Design Overview}
+This section covers an overview of our laser engraver design. This section will discuss the required functionality, the required components for building, the theory of operation, and the potential design alternatives. 
+\subsection{Requirements}
+Our laser engraver shall meet the following functionality requirements:
+\begin{itemize}
+	\item The laser engraver shall be able to etch at least a 100 mm by 100 mm square.
+	\item The laser and x-y actuators shall be mounted within their own secure housing.
+	\item The laser shall be securely attached to the x-y actuators.
+	\item The laser brightness shall be changed for different instruction purposes.
+	\item The laser location shall be able to return a home position with a single instruction.
+	\item The microcontroller shall be able to communicate serially with the PC via USB.
+	\item The program for turning images into function commands shall ????***** I don't know how to word this 
+	\item The microcontroller shall be able to receive instructions manually through a terminal on the PC.
+	\item The microcontroller shall be able to receive instructions automatically from the PC via the image breakdown program. *****
+	\item The microcontroller shall be able to send a response flag via the UART back to the PC upon instruction completion.
+	\item The microcontroller shall be able to send an error flag via the UART back to the PC upon invalid instruction.
+	
+\end{itemize}
+\subsection{Dependencies}
+The following components are required to build the complete laser engraver design:
+\begin{itemize}
```

Comment:
It would maybe be good to split these into categories- Hardware, Software, Electrical?